http.go

```diff
@@ -443,12 +449,17 @@ func errorText(err error) string {
 // response headers into a given io.Writer.
 type HandshakeHeader interface {
 	io.WriterTo
+	Get(key string) string
```

Comment:
Hm, that's a backward incompatibile change. Need to think about that..

Reply:
yes, let me know when you have decided 👍